code/bin/build.bat

```diff
@@ -18,12 +18,26 @@ set custom_bin=%custom_root\bin
 cd %location%
 
 set mode=%1
-if "%mode%" == "" (set mode="/DDEV_BUILD")
+set backend=%1
+
+REM I don't know how to loop over parameters.
```

Comment:
I have spent time to figure this out before, I think you want something like this, but it is very ugly: 
```bat
for %%v in (%*) do call :ParseArg %%v

REM Then at the bottom define the ParseArg function
:ParseArg
  set release_flag=release
  set tracing_flag=tracing
  
  if "%~1"=="%release_flag%" (
    set build_mode=release
  ) else if "%~1"=="%tracing_flag%" (
    set auto_tracing=on
  ) else (
    echo Invalid Argument "%~1"
    EXIT /B 1
  )
EXIT /B 0
```
I just copied this out of one of my other projects so will obviously need changeing.